xml/System.Runtime.Intrinsics.X86/AvxVnni.xml

```diff
@@ -0,0 +1,285 @@
+<Type Name="AvxVnni" FullName="System.Runtime.Intrinsics.X86.AvxVnni">
```

Comment:
The types under here qualify for: https://github.com/dotnet/dotnet-api-docs/issues/6861

If its possible, we should ensure the `RequiresPreviewFeatures` attribute is visible and that the page comes with some sort of disclaimer that the APIs are preview and are subject to change (including being removed) between now and the next release (up until the attribute is removed).